pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "quartz_solar_forecast"
+version = "1.0.69"
```

Comment:
could you update https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/blob/main/.bumpversion.cfg#L7, to be
```
[bumpversion:file:pyproject.toml]
search = version = "{current_version}"
replace = version = "{new_version}"
```